core/container/externalbuilder/copy_test.go

```diff
@@ -19,23 +20,35 @@ import (
 
 var _ = Describe("copy", func() {
 	var (
-		logger                             *flogging.FabricLogger
-		srcRootDir, srcSubDir, destRootDir string
-		srcRootFile, srcSubFile            *os.File
-		err                                error
+		logger                  *flogging.FabricLogger
+		srcRootDir, destRootDir string
+		err                     error
 	)
 
 	BeforeEach(func() {
 		srcRootDir, err = ioutil.TempDir("", "copy-test-")
 		Expect(err).NotTo(HaveOccurred())
-		srcSubDir, err = ioutil.TempDir(srcRootDir, "sub-")
+
```

Comment:
Looks like you've copied a pattern from the original code of making all file permissions 0777.

I'd prefer something more sane like 0644 for regular files (no execute and only writable by owner) and 0755 for directories. Also, the use of `os.ModePerm` hides the actual 0777 value via indirection and does not use the constant as intended - a mask to retrieve the permission bits.